### ISSUE TITLE
Allow changing the font size for the chat view

### ISSUE THREAD

**Issue by enko** [dino/dino#298]
At the moment the chat view on my screen has much whitespace and could benefit from larger font size and larger avatars, as they are hardly recognizable:

![bildschirmfoto_2018-02-27_13-33-18](https://user-images.githubusercontent.com/19407/36728952-cda84fa4-1bc2-11e8-95a1-587aa728f6e2.png)

Please make a setting which allows me to increase avatar and font size.

**Comment by enko** [comment]
larma in the chat suggested setting `GDK_SCALE` to 2 which is quite the improvement, but that has the negative that the roster is too big. larma also suggested setting `GDK_DPI_SCALE` which increases only the font size. I would want something like `GDK_SCALE` but only for for the chat view. The Avatars are still a bit too small in my taste.

**Comment by Echolon** [comment]
This would even increase accessibility for handicapped people

**Comment by dos1** [comment]
Yeah, this actually should be an accessibility feature request that has nothing to do with GDK_SCALE/GDK_DPI_SCALE. Sometimes you just want to increase the font size temporarily, for instance when showing something to a person with poor eyesight.

**Comment by whoizit** [comment]
GDK_SCALE not working for me on wayland (sway). GDK_DPI_SCALE works, but only for font :(

**Comment by whoizit** [comment]
works with xorg backend
`GDK_BACKEND=x11 GDK_SCALE=1.5 dino`

**Comment by jeroenpraat** [comment]
Another (better working) workaround is:

edit `~/.config/gtk-3.0/gtk.css` and add for example:

```css
window.dino-main {
        font-size: 20px;
}

window.dino-main .dino-conversation {
        font-size: 24px;
}
```

And restart Dino.

**Comment by hirusi** [comment]
I found the following settings very comfortable:

```
window.dino-main {
        font-size: 16.5px;
}

window.dino-main .dino-conversation {
        font-size: 17.5px;
}
```

**Comment by eerielili** [comment]
@enko @Echolon could you test this PR [#1495](https://github.com/dino/dino/pull/1495)
i should also do something for avatar size upon zooming
